recipe/meta.yaml

```diff
@@ -12,17 +12,18 @@ source:
 
```

Comment:
v0.6.0 has been released https://github.com/arviz-devs/xarray-einstats/tree/v0.6.0

Reply:
done.

Reply:
It's still 0.5.1

Reply:
done.